### ISSUE TITLE
Evaluation results

### ISSUE THREAD

**Issue by yeachan-kr** [JNAIC/PIPMN#2]
Hi, 

While I'm checking your implementations to replicate the best results in US8k, I found that the current evaluation does not follow the 10-fold cross-validation setup in the US8K original paper which is crucial point in this dataset.

Could you check this out and report the comparable performance to others?

Best,

**Comment by dongdongunique** [comment]
I am sorry for my delay. Could you tell me the results and code of your evaluation? I have repeated many tests on my computer to verify my results with the code of the model, training and evaluation test in this repository.

**Comment by dongdongunique** [comment]
Or could you check my evaluation code to see if there are any problems?  I am very much looking forward to your response and solving your problems.

**Comment by dongdongunique** [comment]
Possible Problems:
- The hyperparameter: process length is too large, which will cause the overfitting(recommendation: 1-10 for U8k )

**Comment by dongdongunique** [comment]
Possible Problem 2:

- Does the training epochs are finished?   The validation accuracy is 96% when the training accuracy is 99%.

**Comment by yeachan-kr** [comment]
In [train_and_eval.py](https://github.com/JNAIC/PIPMN/blob/main/train_and_evaluation/train_and_eval.py), 
`x_train, x_test, y_train, y_test = train_test_split(np.array(x), y_all, test_size=0.1, random_state = random_state)`
is problematic because your code re-shuffles the train and test data rather than using 10-fold validation that is pre-defined in original US-8K dataset.

This is crucial point because each fold is constructed from different sources (please refer the origianl paper for US8K). Moreover, to my knowledge, other baselines evaluate their method on 10-fold settings instead of randomly selecting small portion of data as test data. 

To fairly compare with others, (1) PIPMN should be evaluated on 10-fold settings (2) other baselines should be evaluated on the same settings as PIPMN.

Best,

**Comment by dongdongunique** [comment]
Thanks a lot for your criticism. I will re-evaluate my method under the same setting. And I will keep you informed of the up-to-date results.
